### DISCUSSION TITLE
Next Steps - 2021 Year End Retrospective

### DISCUSSION THREAD

**Discussion by adambkaplan** [shipwright-io/community#51] [Ideas]
## Next Steps

After our year end review, what actions should we take to improve Shipwright?

This can be new features, community process, new practices, organization changes, etc.

For reference, here is the link to the retrospective minutes: https://github.com/shipwright-io/community/issues/49

Please provide your thoughts by January 31, 2022.

**Comment by adambkaplan** [comment]
:bulb: Improvement: a cohesive strategy for our documentation. The goal here is that we structure docs across our repositories such that end-user docs show up on the website, and contributors know where to write docs.

**Comment by qu1queee** [comment]
From my side:

1) This one is not to improve, but rather for avoiding a decline in our best practices. We need a clear direction for the next months(_Roadmap_), similar to what was done for [2021](https://hackmd.io/@imjasonh/HkydtciE_). This should help to align what we do, with our desire state. 

2) We should _value_ user feedback and react as soon as possible. We have feedback across different places where we haven't react on time. 

3) We should honored our release cadence.

4) We should tighten the feedback loop between contributors across _pull request_, _SHIPs_ and _Issues_.

5) We should understand the trends in the industry and find melting points with other technologies. Aiming for a _PoC_, _SHIP_, _PR_ is ideal.  For example this https://github.com/shipwright-io/build/issues/886 .

6) We need more user feedback, therefore we need to expand our reach, e.g. _conferences, blog posts, hands-on-lab, office hours, etc_

7) We should ensure that our documentation is neat.

**Comment by SaschaSchwarze0** [comment]
In addition to https://github.com/shipwright-io/community/issues/49#issuecomment-1014509077.

I would like to take a realistic view on us: we atm have zero engineers with 100 % focus on Shipwright and nothing else. This is not necessarily bad. That's our current reality.

The result is that we often read or hear things like "my review was a bit distracted because of getting this in with a bunch of other multi-tasking I've had to do". We notice that sometimes we promise to look at something but then nothing happens. We observe pull requests that are not getting any feedback for weeks. We observe pull requests that are initially opened but after giving feedback it takes weeks to get things sorted out. Some of these things sometimes cause frustration.

So, I think this is something where we need to improve although it's hard to foresee if SLAs like "pull request reviews will be answered within n days" will work.

Things like release cadance or responding user issues and on-top things like blog posts or labs fall into the same bucket.

What I want is that we make realistic goals. We as a community today are not strong enough to improve on all areas at once. So let's pick some that we think we can realistically improve. This should not just help us in our group but also externally as we should not promise too much that we won't do at the end.
